docs/index.asciidoc

```diff
@@ -50,22 +50,22 @@ connecting to Elasticsearch 7.x.
 You can use this plugin to send your {ls} data to {es-serverless}.
 Some differences to note between {es-serverless} and self-managed {es}:
 
-* Use *API keys* to access {serverless-full} from {ls}. 
+* Use *API keys* to access {serverless-full} from {ls}.
 Any user-based security settings in your <<plugins-outputs-elasticsearch,{es} output plugin>> configuration are ignored and may cause errors.
-* {es-serverless} uses *data streams* and {ref}/data-stream-lifecycle.html[{dlm} ({dlm-init})] instead of {ilm} ({ilm-init}). 
+* {es-serverless} uses *data streams* and {ref}/data-stream-lifecycle.html[{dlm} ({dlm-init})] instead of {ilm} ({ilm-init}).
 Any {ilm-init} settings in your <<plugins-outputs-elasticsearch,{es} output plugin>> configuration are ignored and may cause errors.
 * *{ls} monitoring* is available through the https://github.com/elastic/integrations/blob/main/packages/logstash/_dev/build/docs/README.md[{ls} Integration] in {serverless-docs}/observability/what-is-observability-serverless[Elastic Observability] on {serverless-full}.
 
 .Known issue for {ls} to {es-serverless}
 ****
-The logstash-output-elasticsearch `hosts` setting on {serverless-short} defaults the port to 9200 when omitted. 
+The logstash-output-elasticsearch `hosts` setting on {serverless-short} defaults the port to 9200 when omitted.
 Set the value to port :443 instead.
 ****
 
 For more info on sending data from {ls} to {es-serverless}, check out the {serverless-docs}/elasticsearch/what-is-elasticsearch-serverless[{es-serverless} docs].
 
 [id="plugins-{type}s-{plugin}-ess"]
-==== Hosted {es} Service on Elastic Cloud
+==== {ech}
```

Comment:
```suggestion
==== {ls} to {ech}
```

Reply:
This change makes the Elastic Cloud Hosted heading parallel with the "Logstash to Elasticsearch Serverless" heading.